Sources/LiveViewNative/Views/Controls and Indicators/Menus/Menu.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Menu.swift
+//  
+//
+//  Created by Carson Katri on 1/19/23.
+//
+
+import SwiftUI
+
+struct Menu<R: CustomRegistry>: View {
+    @ObservedElement private var element: ElementNode
+    let context: LiveContext<R>
+    
+    init(element: ElementNode, context: LiveContext<R>) {
+        self.context = context
+    }
+    
+    public var body: some View {
+        SwiftUI.Menu {
+            context.buildChildren(of: element, withTagName: "content", namespace: "menu")
+        } label: {
+            context.buildChildren(of: element, defaultSlotFor: "menu")
+            context.buildChildren(of: element, withTagName: "label", namespace: "menu")
```

Comment:
I think these should be combined into one helper that uses the explicit version if present and the default slot otherwise. I can't think of a reason you'd want both the default and explicit one together